iron-list.html

```diff
@@ -232,7 +232,6 @@
     <style>
       :host {
         display: block;
-        position: relative;
```

Comment:
Do you think if it's possible some users may relying on this and removing it will break their apps?